suite-native/atoms/src/DiscreetText.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+import { LayoutChangeEvent } from 'react-native';
+
+import { useAtom, atom } from 'jotai';
+import { Blur, Canvas, Text as SkiaText, useFont } from '@shopify/react-native-skia';
+
+import { Color, TypographyStyle, typographyStylesBase } from '@trezor/theme';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Text } from './Text';
+import { Box } from './Box';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const satoshiFont = require('../../../packages/theme/fonts/TTSatoshi-Regular.otf');
+
+type DiscreetTextProps = {
+    typography?: TypographyStyle;
+    color?: Color;
+    text: string;
+};
+
+export const isDiscreetModeOn = atom(true);
+
+type DiscreetCanvasProps = {
+    width: number;
+    height: number;
+    fontSize: number;
+    text: string;
+};
+const DiscreetCanvas = ({ width, height, fontSize, text }: DiscreetCanvasProps) => {
+    const font = useFont(satoshiFont, fontSize);
+    if (!font) return null;
+
+    return (
+        <Canvas style={{ height, width }}>
+            <SkiaText x={0} y={fontSize} text={text} font={font} />
+            <Blur blur={15} mode="decal" />
+        </Canvas>
+    );
+};
+
+const textStyle = prepareNativeStyle<{ isDiscreet: boolean }>((_, { isDiscreet }) => ({
+    extend: {
+        condition: isDiscreet,
+        style: {
+            opacity: 0,
+            height: 0,
+        },
+    },
+}));
+
+type DiscreetValueProps = {
+    onSetWidth: (width: number) => void;
+    typography?: TypographyStyle;
+    color?: Color;
+    isDiscreet: boolean;
+    text: string;
+};
+const DiscreetTextValue = ({
```

Comment:
Not sure about using `Discreet` in name. It confusing because it's just normal text. I would prefer name `NormalText` or something like that.